tasks.py

```diff
@@ -99,6 +99,8 @@ def docker_compose(context, command, **kwargs):
         "PYTHON_VER": context.nautobot_chatops.python_ver,
         **kwargs.pop("env", {}),
     }
+
+    compose_command = f'docker compose --project-name {context.nautobot_chatops.project_name} --project-directory "{context.nautobot_chatops.compose_dir}"'
```

Comment:
I don't understand this change, it is just going to be overridden a few lines below when it joins the compose command tokens.

Reply:
What is the difference between the tokens and this? It was a merge conflict, and I am not familiar with command tokens.

Reply:
#212 Replaced `docker-compose` with `docker compose` among other changes. I believe this is why you get the merge conflict.